test/ClientTest.php

```diff
@@ -121,8 +121,9 @@ public function testClientDisconnect() {
 
             $context = (new ClientTlsContext)->withoutPeerVerification();
             $client = new DefaultClient(null, null, $context);
+            $port = parse_url($address, PHP_URL_PORT);
             $promise = $client->request(
-                (new \Amp\Artax\Request("https://$address/", "POST"))->withBody("body")
+                (new \Amp\Artax\Request("https://localhost:$port/", "POST"))->withBody("body")
```

Comment:
Why that?

Reply:
Because of the single vhost == default host not holding anymore and being explicit now.